vscode-dotnet-sdk-extension/src/test/functional/DotnetCoreAcquisitionExtension.test.ts

```diff
@@ -304,49 +304,6 @@ suite('DotnetCoreAcquisitionExtension End to End', function ()
       await vscode.commands.executeCommand('dotnet-sdk.uninstallAll');
     }).timeout(standardTimeoutTime);
 
-    test('Uninstall Command', async () =>
```

Comment:
Is this just an old test or is it no longer valid (or both)?

Reply:
Both!